Makefile

```diff
@@ -29,6 +29,9 @@ test-system: build
 test-heavy: build $(SAMPLEDATA)
 	@go test -v -tags=heavy ./tests
 
+perf-compare: build $(SAMPLEDATA)
```

Comment:
The `.PHONY` list is getting long but it seems correct to make this phony, too. https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Thanks for the education. I've added it to the phony list and re-tested.